pkg/beacon/entry/submission.go

```diff
@@ -99,11 +83,21 @@ func (res *relayEntrySubmitter) submitRelayEntry(
 				}
 			}
```

Comment:
What if have an error from `SubmitRelayEntry` and entry is still in progress? 

Shouldn't we log it?

```
if !isEntryInProgress {
  logger.Infof(
    "[member:%v] relay entry already submitted",
    res.index,
  )
  return nil
}

logger.Errorf(
  "[member:%v] could not submit relay entry: [%v]",
  res.index,
  err,
)
```

Note that we'll have to distinguish between the error returned from `SubmitRelayEntry` and `IsEntryInProgress`.